lib/page/questions/widget/questions_survey_answers_widget.dart

```diff
@@ -56,6 +58,16 @@ class _QuestionsSurveyAnswersWidgetState
           itemCount: widget.question.answers.length,
           onRatingUpdate: (rating) => _saveRatingBarsAnswer(rating),
         );
+      case DisplayType.choice:
+        return _buildMultipleChoicesAnswer(
+          answers: widget.question.answers,
+          onItemsChanged: (items) => _saveMultipleChoicesAnswer(
+            items
+                .map((item) =>
+                    SubmitSurveyAnswerRequest(id: item.id, answer: item.text))
```

Comment:
Same to previous PR, we should not build API request components from presentation layer directly like this 🙏